File: FMDB demo/FMDB demo/ViewController.m
//
//  ViewController.m
//  FMDB demo
//
//  Created by czljcb on 16/3/22.
//  Copyright © 2016年 lQ. All rights reserved.
//

#import "ViewController.h"
#import "FMDBTools.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    NSData *data = [NSKeyedArchiver archivedDataWithRootObject:@{@"name": @"czljcb"}];
    
    [FMDBTools insert:@"222" data:data];
    
    
    
    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event
{
   NSLog(@"%@",[FMDBTools dataList]) ;
}

@end

File: Swift - 属性监听/Swift - 属性监听/ViewController.swift
//
//  ViewController.swift
//  Swift - 属性监听
//
//  Created by czljcb on 16/2/26.
//  Copyright © 2016年 czljcb. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        let p = Person()
        p.name = "json"
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

